src/types.ts

```diff
@@ -187,6 +187,15 @@ export function shouldShowLastObserved(queryType?: QueryType): boolean {
   return queryType === QueryType.PropertyAggregate || queryType === QueryType.PropertyValueHistory;
 }
 
+export function shouldShowOptionsRow(query: SitewiseQuery, showProp: boolean): boolean {
+  const shouldShowLastObservedSwitch =
+    shouldShowLastObserved(query.queryType) && !Boolean(query.propertyAlias) && showProp;
+  const shouldShowWithPropertyAlias =
+    // shouldn't show the row when querying associated assets with property alias, otherwise show it every time property alias is set
+    query.propertyAlias && !isListAssociatedAssetsQuery(query);
```

Comment:
SO I changed the logic (and the UI) a little bit from what we currently have in main. Right now, the Quality, Time and Format rows are not displayed if the query is `queryPropertyValue` and the `propertyAlias` is used. It looks to me like this isn't intended, as this query can also return a time series that gets formatted by all these fields, as seen here: 
https://github.com/grafana/iot-sitewise-datasource/blob/main/pkg/framer/property_value.go#L23

Furthermore, there is an unintended side-effect in the old form where if you select an asset, then add a propertyAlias, the Asset field is hidden, but the formatting options stay because the assetId doesn't get cleared from the query object.
In any case, I think explicitly allowing these fields to be selected with propertyAlias query is the right way to go.